.github/workflows/javascript-lint.yml

```diff
@@ -0,0 +1,44 @@
+name: Javascript lint
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
This will need to be changed for the downstream version by the way and adapted to our release branches, also for other workflows. There is no master branch on the downstream repo.

Reply:
That's a good point, I didn't consider this. What branches do we want to target for this? We could use [the syntax outlined here](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags) to match specific ones.

Reply:
As discussed on chat, I'll target 4.1, and also 4.0 if possible.